webapp/components/PanelRight/PanelRight.js

```diff
@@ -22,7 +22,7 @@ PanelRight.propTypes = {
   children: PropTypes.oneOfType([PropTypes.node, PropTypes.arrayOf(PropTypes.node)]).isRequired,
   header: PropTypes.node,
   onClose: PropTypes.func.isRequired,
-  width: PropTypes.number, // width of the panel in px
+  width: PropTypes.string, // width of the panel (e.g. '1000px' or '90vw')
```

Comment:
should defaultProps must be changed as well to '500px' ?

Reply:
sure, I've forgot it